definitions/tools/pvacseq.cwl

```diff
@@ -4,22 +4,28 @@ cwlVersion: v1.0
 class: CommandLineTool
 label: "run pVACseq"
 
-baseCommand: ["pvacseq", "run"]
+baseCommand: [
+    "ln", "-s"
+]
+arguments: [
+    { valueFrom: "$TMPDIR", shellQuote: false },
+    "/tmp/pvacseq",
+    { valueFrom: " && ", shellQuote: false },
+    "export", "TMPDIR=/tmp/pvacseq",
+    { valueFrom: " && ", shellQuote: false },
```

Comment:
This seems kind of weird.  I'm guessing that the issue is TMPDIR is set in a non-standard way within the cromwell job?

Reply:
Cromwell sets a unique tmpdir for each step that can get very long. Python multi threading will throw an error`AF_UNIX path too long` in that situation.  We think this is thrown only in pVACtools because it uses multithreading via  pymp.

Reply:
See
https://github.com/broadinstitute/cromwell/issues/3647

Reply:
Ugh. yeah, okay, I get it then, and don't see a more elegant way to do it than what you have here!  +1